dashboards/google-dns-query/dns-query.json

```
@@ -0,0 +1,257 @@
+{
+  "displayName": "Cloud DNS",
+  "mosaicLayout": {
+    "columns": 48,
+    "tiles": [
+      {
+        "width": 16,
+        "height": 16,
+        "widget": {
+          "title": " Error Rate per ManagedZone",
```

```suggestion
          "title": "Error Rate per ManagedZone",
```

Let's remove the leading space here